r/rsparkling/rsparkling.Rproj

```diff
@@ -13,4 +13,4 @@ StripTrailingWhitespace: Yes
 BuildType: Package
 PackageUseDevtools: Yes
 PackageInstallArgs: --no-multiarch --with-keep.source
-PackageRoxygenize: rd,collate,namespace
+PackageRoxygenize: rd,collate
```

Comment:
I'm not using it, it make sense only if you want to sync your rstudio settings, quite developer-specific. I can do git rm and add to gitignore.